src/SvgXml.web.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { ViewProps } from 'react-native';
+import { View, unstable_createElement } from 'react-native-web';
+import { XmlProps } from './xml';
+type SvgXmlProps = XmlProps &
+  ViewProps & {
+    height: number | string;
+    width: number | string;
+    viewBox?: string;
+    preserveAspectRatio?: string;
+    color?: string;
+    title?: string;
+  };
+const SvgXml = React.forwardRef<HTMLOrSVGElement, SvgXmlProps>(
+  ({ xml, ...props }, fowardRef) => {
+    const { attributes, innerSVG } = parseSVG(xml);
+    const camelAttributes = kebabToCamel(attributes);
+
+    const svgRef = React.createRef<SVGElement>();
+    React.useLayoutEffect(() => {
+      if (!svgRef.current) {
+        return;
+      }
+      svgRef.current.innerHTML = innerSVG;
+    }, [innerSVG, svgRef]);
+
+    const {
+      height,
+      width,
+      viewBox,
+      preserveAspectRatio,
+      color,
```

Comment:
Same here

Reply:
Here I left only those props that are supported by the web ```svg``` tag